Makefile

```diff
@@ -42,8 +37,7 @@ remotestorage.js: $(SOURCES)
 	$(NPM) run build
 
 doc:
-	mkdir -p $(DOC_DIR) $(DOC_CONFIG_DIR)
-	$(DOC_BIN) $(DOC_INPUTS) -o html $(DOC_DIR) -p $(DOC_CONFIG_DIR) -s Default $(DOC_CUSTOM_CSS)
+	cd doc && sphinx-build . _build/html
```

Comment:
Wouldn't we have to also `cd` back out here? (Haven't tested it, and am not a Make expert.) If so, I think we should rather use relative paths to the root dir for the arguments.

Reply:
Good point, I checked and Make invokes a new sub-shell for each line (https://www.gnu.org/software/make/manual/html_node/Execution.html#Execution). At first I tried to use relative paths but I couldn't make it work:

```
$ sphinx-build doc doc/_build/html
Running Sphinx v1.6.4
loading pickled environment... done

Exception occurred:
  File "/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
The full traceback has been saved in /var/folders/85/4fncgmr946v2mt7p58hkz6tc0000gn/T/sphinx-err-whlqM5.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

Reply:
Are you saying we don't have to return then (sounds like it to me)? It'd be easier if you confirmed that explicitly, so we don't have to read the same docs as well.

Reply:
We don't have to return